playground/compound-components/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
```

Comment:
[non-blocking] maybe merge this playground to `playground/react` to reduce the number of playgrounds?